Utils/test_upload_flow/create_test_branch.py

```diff
@@ -153,37 +327,33 @@ def parse_arguments() -> argparse.Namespace:
         repo.git.checkout(original_branch)
     else:
         original_branch = repo.active_branch
+
     try:
-        new_branch_name = f"{original_branch}_upload_test_branch_{repo.active_branch.object.hexsha}"
+        new_branch_name = args.test_branch if args.test_branch else f"{original_branch}_upload_test_branch_{time.time()}"
         content_path = Path(__file__).parent.parent.parent
         packs_path = content_path / 'Packs'
         branch = create_new_branch(repo, new_branch_name)
 
-        new_pack_path = create_new_pack()
-        add_dependency(packs_path / 'Armis', new_pack_path)
-        enhance_release_notes(packs_path / 'ZeroFox')
-        change_image(packs_path / 'Armis')
-
-        update_existing_release_notes(packs_path / 'Box', "2_1_2.md")
-        enhance_release_notes(packs_path / 'Box')
-        update_existing_release_notes(packs_path / 'Base', "1_13_13.md")
-        add_1_0_0_release_notes(packs_path / 'BPA')
-        set_pack_hidden(packs_path / 'Microsoft365Defender')
-        update_readme(packs_path / 'Maltiverse')
-        update_pack_ignore(packs_path / 'MISP')
-
-        add_pack_to_landing_page('Trello')
+        logging.info(f"Starts doing test changes on branch '{branch.name}'")
+        do_changes_on_branch(packs_path)
 
         for p in changed_packs:
             repo.git.add(f"{p}/*")
 
-        repo.git.commit(m="Added Test file")
-        repo.git.push('--set-upstream', 'https://code.pan.run/xsoar/content.git', branch)  # disable-secrets-detection
+        repo.git.commit(m="Added Test file", no_verify=True)
+        repo.git.push('--set-upstream',
```

Comment:
when do we delete the remote branch?

Reply:
There is a job that runs every night and deletes branches that are only in gitlab and not in github